test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -44,6 +44,10 @@ class test_uct_sockaddr : public uct_test {
     }
 
     void init() {
+        if (!strcmp(GetParam()->md_name.c_str(), "tcp")) {
+            UCS_TEST_SKIP_R("tcp does not support sockaddr over iface");
```

Comment:
message looks confusing

Reply:
is this better?
""tcp does not support sockaddr without a cm""

Reply:
need to check md flags to disqualify tcp_sockcm here